features/ink-enhancement.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-09-09
-name: Ink API
-description: TODO
+name: Ink enhancement
+description: The Ink API allows browsers to directly make use of available OS-level compositors when drawing pen strokes in an inking app feature, thereby reducing latency and increasing performance.
```

Comment:
A heads up: if you're going to copy and paste from MDN, you're pretty much guaranteeing at least one round of revisions. Perversely, MDN has a hard time briefly describing web APIs in a developer-oriented way. You might want to read through our existing descriptions and [the guidelines](https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md#descriptions) to get a better handle on what these things need to say.

OK, onward. What I'm aiming to do here:

- Name the entry point
- Actively describe what happens—what the developer gets—by invoking this API

```suggestion
description: The `navigator.ink` API draws between pointer events, reducing the delay between moving a stylus or cursor and a pen stroke appearing on screen.
```